### PR TITLE
[Storybook]Create components for Topbar and Toolbars #293

### PR DESCRIPTION
**Pull request by Sarfraz-droid**
Create components for Topbar and Toolbars from issue #293 

## Screenshots or GIF (In case of UI changes):
Issue293Storybook2

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
-    .register("/firebase-messaging-sw.js")
-    .catch(function(err) {
-      console.log("Service worker registration failed, error:", err);
+    .register('/firebase-messaging-sw.js')
+    .catch(function (err) {
+      console.log('Service worker registration failed, error:', err);
     });
 }
 
+const theme = createTheme({
```
Comment (Niweera) [inline src/index.js:19]: is this `theme` variable used in the code? can you double check?

Comment (Sarfraz-droid) [reply src/index.js]: Sorry this isn't used in the app currently. I added this to test the theme in the website

**Review by Niweera — APPROVED** [review]

**Review by AsithaIndrajith — APPROVED** [review]

File: src/components/NavBar/_old/MiniNavbar/index.js [hunk 2]
```diff
@@ -41,29 +41,29 @@ const MiniNavbar = ({ type }) => {
...
                     type={type && type === "/login" ? "primary" : "link"}
-                    variant="outlined"
+                    variant='outlined'
                   >
                     <Link to={"/login"}>Log In</Link>
                   </Button>
 
                   <Button
-                    key="1"
+                    key='1'
```
Comment (shehand) [inline src/components/NavBar/_old/MiniNavbar/index.js:66]: Revert these changes. Do not use single quotes.

File: src/components/NavBar/index.js [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-import { useAuthStatus } from "../../helpers/customHooks";
-import { useSelector } from "react-redux";
+import React, { useState, useEffect } from 'react';
+import { useLocation, useHistory } from 'react-router-dom';
+import navbarPaths from './navbarPaths';
+import { useAuthStatus } from '../../helpers/customHooks';
+import { useSelector } from 'react-redux';
+import MainNavbar from './new/MainNavbar';
+import MiniNavbar from './new/MiniNavbar';
+import { useFirebase } from 'react-redux-firebase';
```
Comment (shehand) [inline src/components/NavBar/index.js:8]: Use double quotes. Please use prettier or some kind of beautifier.

File: src/components/NavBar/new/MainNavbar/LeftMenu.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import { NavLink, useLocation } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import { useAllowDashboard } from '../../../../helpers/customHooks';
+import useGetPermissions from '../../../../helpers/customHooks/useGetPermissions';
+import AppsIcon from '@material-ui/icons/Apps';
+import { IconButton, Menu } from '@material-ui/core';
+const LeftMenu = ({ mode }) => {
+  const permissions = useGetPermissions();
```
Comment (shehand) [inline src/components/NavBar/new/MainNavbar/LeftMenu.js:11]: Same comment as above.

File: src/components/NavBar/new/MainNavbar/RightMenu.js [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+import SettingsOutlinedIcon from '@material-ui/icons/SettingsOutlined';
+import CodeOutlinedIcon from '@material-ui/icons/CodeOutlined';
+import ExitToAppOutlinedIcon from '@material-ui/icons/ExitToAppOutlined';
+import PersonOutlineOutlinedIcon from '@material-ui/icons/PersonOutlineOutlined';
+import { avatarName } from '../../../../helpers/avatarName';
+import { Link } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import Grid from '@material-ui/core/Grid';
+import Menu from '@material-ui/core/Menu';
+import { makeStyles } from '@material-ui/core/styles';
```
Comment (shehand) [inline src/components/NavBar/new/MainNavbar/RightMenu.js:17]: Double quotes.

**Review by shehand — COMMENTED** [review]
Please use some kind of beautifier.

**Review by Niweera — APPROVED** [review]
LGTM!
